testng jar is not downloaded for local testing

In local testing, set
```
export TEST_JDK_HOME=xxxx
export BUILD_LIST=functional
export JDK_VERSION=17
```

`make compile` failed with the following msg:
```
make -f clean.mk cleanBuild
make[1]: Entering directory '/root/hostdir/testDirectory/aqa-tests/TKG'
rm -f -r /root/hostdir/testDirectory/aqa-tests/TKG/../../jvmtest
make[1]: Leaving directory '/root/hostdir/testDirectory/aqa-tests/TKG'
make -f compile.mk compile
make[1]: Entering directory '/root/hostdir/testDirectory/aqa-tests/TKG'
rm -f -r /root/hostdir/testDirectory/aqa-tests/TKG/../TKG/output_compilation; \
mkdir -p /root/hostdir/testDirectory/aqa-tests/TKG/../TKG/output_compilation; \
(ant -f scripts/build_test.xml -DTEST_ROOT=/root/hostdir/testDirectory/aqa-tests/TKG/.. -DBUILD_ROOT=/root/hostdir/testDirectory/aqa-tests/TKG/../../jvmtest -DJDK_VERSION=17 -DJDK_IMPL=openj9 -DJCL_VERSION=latest -DBUILD_LIST=functional -DRESOURCES_DIR=/root/hostdir/testDirectory/aqa-tests/TKG/../../jvmtest/TestConfig/resources -DSPEC=linux_ppc-64_le -DTEST_JDK_HOME=/root/hostdir/openj9-openjdk-jdk17/build/linux-ppc64le-server-release/images/jdk -DJVM_VERSION=openjdk17-openj9 -DLIB_DIR=/root/hostdir/testDirectory/aqa-tests/TKG/../TKG/lib 2>&1; echo $? ) | tee "/root/hostdir/testDirectory/aqa-tests/TKG/../TKG/output_compilation/compilation.log"; \
if [ -z "$(tail -1 /root/hostdir/testDirectory/aqa-tests/TKG/../TKG/output_compilation/compilation.log | grep 0)" ]; then perl scripts/moveDmp.pl --compileLogPath="/root/hostdir/testDirectory/aqa-tests/TKG/../TKG/output_compilation/compilation.log" --testRoot="/root/hostdir/testDirectory/aqa-tests/TKG/.."; false; else rm -f -r "/root/hostdir/testDirectory/aqa-tests/TKG/../TKG/output_compilation"; fi
Buildfile: /root/hostdir/testDirectory/aqa-tests/TKG/scripts/build_test.xml
     [echo]  build.list is TestConfig/build.xml,functional/build.xml

-create_test_directory:
    [mkdir] Created dir: /root/hostdir/testDirectory/jvmtest

stage_test_material:
     [copy] Warning: TKG/output_compilation/compilation.log modified in the future.
     [copy] Copying 96 files to /root/hostdir/testDirectory/jvmtest

build:

init:
    [mkdir] Created dir: /root/hostdir/testDirectory/jvmtest/TestConfig

build:
     [copy] Copying 10 files to /root/hostdir/testDirectory/jvmtest/TestConfig
     [copy] Copied 4 empty directories to 2 empty directories under /root/hostdir/testDirectory/jvmtest/TestConfig
     [copy] Copying 1 file to /root/hostdir/testDirectory/jvmtest/TestConfig

dist_functional:
    [mkdir] Created dir: /root/hostdir/testDirectory/jvmtest/functional
     [copy] Copying 1 file to /root/hostdir/testDirectory/jvmtest/functional

build_functional:

buildCmdLineTestUtils:

check-jar:

build:

init:
    [mkdir] Created dir: /root/hostdir/testDirectory/jvmtest/functional/cmdLineTests/utils

compile:
     [echo] Ant version is Apache Ant(TM) version 1.10.7 compiled on October 24 2019
     [echo] ============COMPILER SETTINGS============
     [echo] ===fork:                         yes
     [echo] ===executable:                   /root/hostdir/openj9-openjdk-jdk17/build/linux-ppc64le-server-release/images/jdk/bin/javac
     [echo] ===debug:                        on
     [echo] ===destdir:                      /root/hostdir/testDirectory/aqa-tests/TKG/../../jvmtest/functional/cmdLineTests/utils
    [javac] Compiling 11 source files to /root/hostdir/testDirectory/aqa-tests/functional/cmdLineTests/utils/bin
    [javac] /root/hostdir/testDirectory/aqa-tests/functional/UnsafeAccess/src_170/org/openj9/test/access/UnsafeClasses.java:30: error: package org.testng does not exist
    [javac] import org.testng.Assert;
    [javac]                  ^
    [javac] /root/hostdir/testDirectory/aqa-tests/functional/UnsafeAccess/src_170/org/openj9/test/access/UnsafeClasses.java:40: error: cannot find symbol
    [javac]             Assert.fail("Unsafe.defineAnonymousClass() is not available in Java 17+");
    [javac]             ^
    [javac]   symbol:   variable Assert
    [javac]   location: class UnsafeClasses
    [javac] Note: /root/hostdir/testDirectory/aqa-tests/functional/cmdLineTests/utils/src_shared/org/openj9/test/classtests/ClassTestHelper.java uses unchecked or unsafe operations.
    [javac] Note: Recompile with -Xlint:unchecked for details.
    [javac] 2 errors

BUILD FAILED
/root/hostdir/testDirectory/aqa-tests/TKG/scripts/build_test.xml:76: The following error occurred while executing this line:
/root/hostdir/testDirectory/aqa-tests/functional/build.xml:52: The following error occurred while executing this line:
/root/hostdir/testDirectory/aqa-tests/functional/cmdLineTests/buildTools.xml:34: The following error occurred while executing this line:
/root/hostdir/testDirectory/aqa-tests/functional/cmdLineTests/utils/build.xml:114: The following error occurred while executing this line:
/root/hostdir/testDirectory/aqa-tests/functional/cmdLineTests/utils/build.xml:74: Compile failed; see the compiler error output for details.

Total time: 4 seconds
1
make[1]: *** [compile.mk:32: compile] Error 1
make[1]: Leaving directory '/root/hostdir/testDirectory/aqa-tests/TKG'
make: *** [makefile:60: compile] Error 2
```

I think the reason is that there is no getDependentLibs in https://github.com/eclipse-openj9/openj9/blob/master/test/functional/cmdLineTests/utils/build.xml

--- a/test/functional/cmdLineTests/buildTools.xml
+++ b/test/functional/cmdLineTests/buildTools.xml
@@ -22,7 +22,7 @@
   SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 -->
 
-<project name="cmdLineTests" default="buildCmdLineTestTools" basedir=".">
+<project name="cmdLineTestTools" default="buildCmdLineTestTools" basedir=".">
 	<target name="buildCmdLineTestTools" description="build command line test tools">
 		<subant target="build">
 			<fileset dir="${TEST_ROOT}/functional/cmdline_options_tester/" includes="build.xml" />

--- a/test/functional/cmdLineTests/utils/build.xml
+++ b/test/functional/cmdLineTests/utils/build.xml
@@ -34,13 +34,15 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-excepti
 	<property name="src" location="src" />
 	<property name="src_shared" location="src_shared" />
 	<property name="build" location="bin" />
+	<property name="LIB" value="testng" />
+	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml" />
 
 	<target name="init">
 		<mkdir dir="${DEST}" />
 		<mkdir dir="${build}" />
 	</target>
 
-	<target name="compile" depends="init" description="Using java ${JDK_VERSION} to compile the source">
+	<target name="compile" depends="init,getDependentLibs" description="Using java ${JDK_VERSION} to compile the source">
 		<echo>Ant version is ${ant.version}</echo>
 		<echo>============COMPILER SETTINGS============</echo>
 		<echo>===fork:                         yes</echo>

